videoframes.py

```diff
@@ -7,6 +7,7 @@
 
 from constants import FRAME_SELECT_STYLE, SLIDER_MAX_VALUE
 from containers import LiveStreamContainer, RewindedStreamContainer
+from utils import OS
```

Comment:
Since utils is not exported correctly, this does not import anything. This can be fixed with `from utils.OS import OS` or by exporting correctly from the utils package.